### PR TITLE
add installed from playstore check before asking rating

### LINKED ISSUE #299: Ask users to rate on Play store only for Google Play build
As of now we have a single build flavor for Google Play as well as for F-Droid so we are also asking F-droid users also to rate on Google Play.
We should create separate build flavor for Google Play and include it only in that.

Not a high priority but it is the right thing to do.

### PR DESCRIPTION
**Pull request by spuday90**
closes #299

### CODE REVIEW

File: app/src/main/java/com/parishod/watomatic/fragment/MainFragment.java [hunk 1]
```diff
@@ -273,6 +275,18 @@ public void onClick(View v) {
         }
     }
 
+    //REF: https://stackoverflow.com/questions/37539949/detect-if-an-app-is-installed-from-play-store
```
Comment (adeekshith) [inline app/src/main/java/com/parishod/watomatic/fragment/MainFragment.java:278]: Should be moved to a utility class if possible.
Possibly in this PR or later if not.

**Comment by adeekshith** [comment]
Looks good but did not test.
@spuday90 were you able to test this anyway?

File: app/src/main/java/com/parishod/watomatic/fragment/MainFragment.java (hunk 1 above)
Comment (spuday90) [reply app/src/main/java/com/parishod/watomatic/fragment/MainFragment.java:278]: we have few things to move to utilities we'll do all of them in separate issue

**Comment by spuday90** [comment]
> Looks good but did not test.
> @spuday90 were you able to test this anyway?

Yes, if i download apk and install it was giving some other pacakge name, also searched on net almost everyone said playstore package name is same as we added, so it should work

**Review by adeekshith — APPROVED** [review]

**Pull request merged**